

<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="../../">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Navigation &mdash; Pantheon 0.0.2 documentation</title>
      <link rel="stylesheet" type="text/css" href="../../_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="../../_static/css/theme.css?v=e59714d7" />

  
      <script src="../../_static/jquery.js?v=5d32c60e"></script>
      <script src="../../_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
      <script src="../../_static/documentation_options.js?v=e3a6060d"></script>
      <script src="../../_static/doctools.js?v=9a2dae69"></script>
      <script src="../../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../../_static/js/theme.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Preprocessing with scanpy" href="../Preprocessing%20with%20scanpy.html" />
    <link rel="prev" title="Tutorials" href="../index.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="../../index.html" class="icon icon-home">
            Pantheon
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../Download.html">Download</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">Tutorials</a><ul class="current">
<li class="toctree-l2 current"><a class="current reference internal" href="#">Navigation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#overview">Overview</a></li>
<li class="toctree-l3"><a class="reference internal" href="#toolbox">Toolbox</a></li>
<li class="toctree-l3"><a class="reference internal" href="#map-editor">Map Editor</a></li>
<li class="toctree-l3"><a class="reference internal" href="#plot-editor">Plot Editor</a></li>
<li class="toctree-l3"><a class="reference internal" href="#export-results">Export Results</a></li>
<li class="toctree-l3"><a class="reference internal" href="#cluster-group-editor">Cluster Group Editor</a></li>
<li class="toctree-l3"><a class="reference internal" href="#cluster-options">Cluster Options</a></li>
<li class="toctree-l3"><a class="reference internal" href="#highlight-gene">Highlight Gene</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../Preprocessing%20with%20scanpy.html">Preprocessing with scanpy</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Clustering%20with%20sacnpy.html">Clustering with sacnpy</a></li>
</ul>
</li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">Pantheon</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="../../index.html" class="icon icon-home" aria-label="Home"></a></li>
          <li class="breadcrumb-item"><a href="../index.html">Tutorials</a></li>
      <li class="breadcrumb-item active">Navigation</li>
      <li class="wy-breadcrumbs-aside">
            <a href="../../_sources/Tutorials/Navigation/contents.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="navigation">
<h1>Navigation<a class="headerlink" href="#navigation" title="Link to this heading"></a></h1>
<section id="overview">
<h2>Overview<a class="headerlink" href="#overview" title="Link to this heading"></a></h2>
<p>This is an example of PBMC_3k scRNA-seq data labeled by leiden and showed with umap.</p>
<a class="reference internal image-reference" href="../../_images/1overview0.png"><img alt="../../_images/1overview0.png" class="align-center" src="../../_images/1overview0.png" style="width: 1000px;" />
</a>
</section>
<section id="toolbox">
<h2>Toolbox<a class="headerlink" href="#toolbox" title="Link to this heading"></a></h2>
<p>The Toolbox is just on the right of main plot.</p>
<a class="reference internal image-reference" href="../../_images/2mouse1.png"><img alt="../../_images/2mouse1.png" class="align-center" src="../../_images/2mouse1.png" style="width: 600px;" />
</a>
</section>
<section id="map-editor">
<h2>Map Editor<a class="headerlink" href="#map-editor" title="Link to this heading"></a></h2>
<p>Click to choose the map.</p>
<a class="reference internal image-reference" href="../../_images/3map.png"><img alt="../../_images/3map.png" class="align-center" src="../../_images/3map.png" style="width: 600px;" />
</a>
<p>The option of map is decided by the existed view. If there is not existed view of data, the plot map will be constructed by the feature, which is named by <strong>generic_columns</strong>.</p>
<p>Then input the x and y axis.</p>
<a class="reference internal image-reference" href="../../_images/3map1.png"><img alt="../../_images/3map1.png" class="align-center" src="../../_images/3map1.png" style="width: 600px;" />
</a>
</section>
<section id="plot-editor">
<h2>Plot Editor<a class="headerlink" href="#plot-editor" title="Link to this heading"></a></h2>
<p>Check <strong>Log-scaled axis</strong> if log scaled feature is expected to show.</p>
<a class="reference internal image-reference" href="../../_images/3map2.png"><img alt="../../_images/3map2.png" class="align-center" src="../../_images/3map2.png" style="width: 600px;" />
</a>
<p>Select current color with color selection button.</p>
<a class="reference internal image-reference" href="../../_images/3map3.png"><img alt="../../_images/3map3.png" class="align-center" src="../../_images/3map3.png" style="width: 600px;" />
</a>
<p><strong>Gate</strong>: click <em>Gate</em> button, then only the selected dots will show in the plot.
<strong>Trim</strong>: click <em>Trim</em> button, then selected dots will be removed in the plot.
<strong>Show All</strong>: click <em>Show All</em> button, then all the dots will show in the plot.</p>
<a class="reference internal image-reference" href="../../_images/4plot1.png"><img alt="../../_images/4plot1.png" class="align-center" src="../../_images/4plot1.png" style="width: 600px;" />
</a>
</section>
<section id="export-results">
<h2>Export Results<a class="headerlink" href="#export-results" title="Link to this heading"></a></h2>
<p>The exported results include one anndata file (in form of ‘.h5ad’) and a file to save result of clustering, whose head is name of the cluster group.</p>
</section>
<section id="cluster-group-editor">
<h2>Cluster Group Editor<a class="headerlink" href="#cluster-group-editor" title="Link to this heading"></a></h2>
<p>Firstly <em>select the cluster group</em> in the dropdown list.
If there is not existed cluster group, input the group name and click the <strong>Create Group</strong> button to create a new group. (The default group name will be construction of name of x- and y- axis).
Users can also edit the selected group by <strong>Rename Group</strong> and <strong>Delete Group</strong> buttons.</p>
<a class="reference internal image-reference" href="../../_images/5group1.png"><img alt="../../_images/5group1.png" class="align-center" src="../../_images/5group1.png" style="width: 600px;" />
</a>
</section>
<section id="cluster-options">
<h2>Cluster Options<a class="headerlink" href="#cluster-options" title="Link to this heading"></a></h2>
<p>Select dots, input the name of this cluater, and click <strong>Create Cluster</strong> button. Then the created cluster will be shown in the followed checkbox list.</p>
<a class="reference internal image-reference" href="../../_images/6cluster1.png"><img alt="../../_images/6cluster1.png" class="align-center" src="../../_images/6cluster1.png" style="width: 600px;" />
</a>
<p>The <strong>Select Cluster</strong> button will highlight the dots of the checked clusters in the checkbox list.</p>
<p>Click <strong>Add to</strong> button, the selected dots will be added to the checked cluster.</p>
<p>Click <strong>Remove from</strong> button, the selected dots will be removed from the checked cluster.</p>
<p>Click <strong>Update Cluster</strong> button to easily update the checked cluster with the selected dots.</p>
<p>Input the new name of cluster and click <strong>Rename Cluster</strong> to change name of checked cluster.</p>
<p><strong>Merge Cluster</strong> button merges the checked clusters into one cluster(defaultly merged into the first checked cluster).</p>
<p>Click the <strong>Delete Cluster</strong> will delete the checked cluster.</p>
<a class="reference internal image-reference" href="../../_images/6cluster2.png"><img alt="../../_images/6cluster2.png" class="align-center" src="../../_images/6cluster2.png" style="width: 600px;" />
</a>
</section>
<section id="highlight-gene">
<h2>Highlight Gene<a class="headerlink" href="#highlight-gene" title="Link to this heading"></a></h2>
<p>Choose the <strong>Highlight Gene</strong> panel to show the level of gene expression.</p>
<p>Input the gene to analyse, click the <strong>Show Highlight Gene</strong> button, then the degree of gene expression will show in plot.</p>
<p>Users can also import marker list(in form of .csv file) to define marker gene expression leve of certain celltype.</p>
<p>With the <strong>Filter</strong> button, only the cells meet the conditions of hightlight gene will be highlighted in the plot.</p>
<p>Click the <strong>Change Selected</strong> button, the selected dots in the Highlight Gene panel will Synchronously show in the main panel for further options.</p>
<a class="reference internal image-reference" href="../../_images/7hlg1.png"><img alt="../../_images/7hlg1.png" class="align-center" src="../../_images/7hlg1.png" style="width: 800px;" />
</a>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="../index.html" class="btn btn-neutral float-left" title="Tutorials" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="../Preprocessing%20with%20scanpy.html" class="btn btn-neutral float-right" title="Preprocessing with scanpy" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, xinzhu.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>